package.json

```diff
@@ -1,19 +1,21 @@
 {
   "name": "jest-vue-preprocessor",
-  "version": "0.2.0",
+  "version": "0.2.0-development",
```

Comment:
version should start from `1.0.0` if you think it's still beta we should release it as `1.0.0-beta` which may not be possible with `semantic-release`